### PR TITLE
Additions to Polish Translation

### PR DESCRIPTION
**Pull request by Keewayz**
**When merged this pull request will:**
- Describe what this pull request will do
- Each change in a separate line
- Include documentation if applicable
- Respect the [Development Guidelines](https://ace3mod.com/wiki/development/)

### CODE REVIEW

File: addons/cookoff/stringtable.xml [hunk 1]
```diff
@@ -34,6 +35,7 @@
             <French>Modifie la gestion des dégâts pour l'auto-inflammation et les effets d'explosion de tourelle.</French>
             <Chinese>更改殉爆以及炮塔爆炸之傷害控制</Chinese>
             <Czech>Změní poškození ovládání a efekty výbuchu veže</Czech>
+            <Polish>Zmienia obsługę obrażeń podczas cook off i eksplozji wieży</Polish>
```
Comment (veteran29) [inline addons/cookoff/stringtable.xml]:
```suggestion
            <Polish>Zmienia obsługę obrażeń podczas samozapłonu i eksplozji wieży</Polish>
```

File: addons/csw/stringtable.xml [hunk 2]
```diff
@@ -86,6 +89,7 @@
             <Japanese>アドバンスド設置</Japanese>
             <Chinese>進階組裝</Chinese>
             <Czech>Pokročilé skládání</Czech>
+            <Polish>Zaawansowane skladanie</Polish>
```
Comment (veteran29) [inline addons/csw/stringtable.xml]:
```suggestion
            <Polish>Zaawansowane składanie</Polish>
```

File: addons/csw/stringtable.xml [hunk 3]
```diff
@@ -174,6 +188,7 @@
             <Japanese>火器を乗せる</Japanese>
             <Chinese>裝載武器</Chinese>
             <Czech>Připevnit zbraň</Czech>
+            <Polish>Zamonuj Broń</Polish>
```
Comment (veteran29) [inline addons/csw/stringtable.xml]:
```suggestion
            <Polish>Zamontuj Broń</Polish>
```

File: addons/medical_blood/stringtable.xml [hunk 4]
```diff
@@ -64,6 +67,7 @@
             <Portuguese>Duração do Sangue</Portuguese>
             <Chinese>血跡時長</Chinese>
             <Czech>Životnost krve</Czech>
+            <Polish>Żywotność śladów krwi</Polish>
```
Comment (veteran29) [inline addons/medical_blood/stringtable.xml]:
```suggestion
            <Polish>Czas trwania śladów krwi</Polish>
```

File: addons/medical_blood/stringtable.xml [hunk 5]
```diff
@@ -73,6 +77,7 @@
             <Portuguese>Controla o tempo de vida que um objeto de gota de sangue tem.</Portuguese>
             <Chinese>控制血跡在地上的時長</Chinese>
             <Czech>Nastavuje jak dlouho objekty krve na zemi vydrží.</Czech>
+            <Polish>Kontroluje żywotność śladów krwi.</Polish>
```
Comment (veteran29) [inline addons/medical_blood/stringtable.xml]:
```suggestion
            <Polish>Kontroluje czas trwania śladów krwi.</Polish>
```

File: addons/medical_feedback/stringtable.xml [hunk 6]
```diff
@@ -77,6 +81,7 @@
             <Chinese>只有在強烈疼痛時使用</Chinese>
             <French>Juste un fort effet douloureux</French>
             <Czech>Pouze efekt vysoké bolesti</Czech>
+            <Polish>Tylko wysoki efekt bólu</Polish>
```
Comment (veteran29) [inline addons/medical_feedback/stringtable.xml]:
```suggestion
            <Polish>Tylko efekt mocnego bólu</Polish>
```

File: addons/medical_gui/stringtable.xml [hunk 7]
```diff
@@ -915,14 +930,15 @@
...
+            <Polish>Stracił krytyczną ilość krwi</Polish>
         </Key>
         <Key ID="STR_ACE_Medical_GUI_STATUS_TOURNIQUET_APPLIED">
             <English>Tourniquet [CAT]</English>
             <German>Tourniquet [CAT]</German>
             <Russian>Жгут</Russian>
             <Spanish>Torniquete [CAT]</Spanish>
             <French>Garrot [CAT]</French>
-            <Polish>Opaska uciskowa [CAT]</Polish>
+            <Polish>Staza Taktyczna[CAT]</Polish>
```
Comment (veteran29) [inline addons/medical_gui/stringtable.xml]:
```suggestion
            <Polish>Staza Taktyczna [CAT]</Polish>
```

File: addons/medical_statemachine/stringtable.xml [hunk 8]
```diff
@@ -26,6 +28,7 @@
             <Chinese>控制當玩家受致命傷時是否能救起。致命傷是指對頭部或身體造成可觀傷害所造成的。</Chinese>
             <French>Définit si les joueurs décèdent en cas de blessure mortelle. Une blessure mortelle est causée par un dommage important à la tête ou au torse.</French>
             <Czech>Nastavuje zda hráči mohou utrpět smrtelné zranění. Smrtelné zranění je utrpěto významným poškozením hlavy nebo těla.</Czech>
+            <Polish>Definiuje, kiedy gracze mogą otrzymać śmiertelne obrażenia. Śmiertelne obrażenia są spowodowane znacznym uszkodzeniem głowy lub ciała.</Polish>
```
Comment (veteran29) [inline addons/medical_statemachine/stringtable.xml]:
```suggestion
            <Polish>Definiuje, kiedy gracze mogą otrzymać śmiertelne obrażenia. Śmiertelne obrażenia są spowodowane znacznym uszkodzeniem głowy lub torsu.</Polish>
```

File: addons/medical_statemachine/stringtable.xml [hunk 9]
```diff
@@ -42,6 +46,7 @@
...
             <French>Définit si les unités IA décèdent en cas de blessure mortelle. Une blessure mortelle est causée par un dommage important à la tête ou au torse.\nSi réglé sur "Toujours", cela produit effectivement un comportement de "Mort instantanée" car les unités IA mourront immédiatement de toute blessure mortelle.\nNOTE : Tout mode autre que "Toujours" nécessite l'activation de l'option "Inconscience IA".</French>
             <Czech>Nastavuje zda AI může utrpět smrtelné zranění. Smrtelné zranění je utrpěto významným poškozením hlavy nebo těla.\nPokud je tato možnost nastavena na "Vždy", efektivně nastává stav "Povolit okamžitou smrt AI", protože AI okamžitě zemře při jakémkoliv smrtelném zranění.\nPOZNÁMKA: Jakýkoliv jiný stav než "Vždy" potřebuje zapnutou možnost "Bezvědomí AI".</Czech>
+            <Polish>Definiuje, kiedy AI może otrzymać śmiertelne obrażenia. Śmiertelne obrażenia są spowodowane znacznym uszkodzeniem głowy lub ciała.\n Ustawienie „Zawsze” powoduje „Natychmiastową śmierć AI”, ponieważ AI natychmiast umiera z powodu śmiertelnych obrażeń.\n UWAGA: Każdy inny tryb niż „Zawsze" wymaga włączenia nieprzytomności AI.</Polish>
```
Comment (veteran29) [inline addons/medical_statemachine/stringtable.xml]:
```suggestion
            <Polish>Definiuje, kiedy AI może otrzymać śmiertelne obrażenia. Śmiertelne obrażenia są spowodowane znacznym uszkodzeniem głowy lub torsu.\n Ustawienie "Zawsze" powoduje "Natychmiastową śmierć AI", ponieważ AI natychmiast umiera z powodu śmiertelnych obrażeń.\n UWAGA: Każdy inny tryb niż "Zawsze" wymaga włączenia nieprzytomności AI.</Polish>
```

File: addons/medical_statemachine/stringtable.xml [hunk 10]
```diff
@@ -66,6 +71,7 @@
...
             <Czech>Nastavuje zda AI může upadnout do bezvědomí namísto okamžité smrti.\nToto nastavení funguje společně s "Smrtelná zranění AI" protože srdeční zástava potřebuje možnost upadnout do bezvědomí.\nTyto možnosti jsou separované, protože jednotky mohou upadnout do bezvědomí kvůli kritickému stavu způsobenému ne smrtelnými zraněními.\nV podstatě to znamená, že pokud chcete zapnout srdeční zástavu pro AI, tato možnost musí být zapnutá.</Czech>
+            <Polish>Definiuje, czy AI może stracić przytomność zamiast natychmiast zginąć.\n Ta opcja działa razem z ustawieniem „Śmiertelne urazy AI”, ponieważ przejście do zatrzymania akcji serca wymaga, aby jednostka mogła stracić przytomność.\n Jednak te ustawienia są rozdzielone, ponieważ jednostki mogą stracić przytomność z powodu krytycznych czynności życiowych powstałych w wyniku urazów innych niż śmiertelne.\n W istocie oznacza to, że aby umożliwić zatrzymanie akcji serca dla jednostek AI, to ustawienie musi być włączone.</Polish>
```
Comment (veteran29) [inline addons/medical_statemachine/stringtable.xml]:
```suggestion
            <Polish>Definiuje, czy AI może stracić przytomność zamiast natychmiast zginąć.\n Ta opcja działa razem z ustawieniem "Śmiertelne urazy AI", ponieważ przejście do zatrzymania akcji serca wymaga, aby jednostka mogła stracić przytomność.\n Jednak te ustawienia są rozdzielone, ponieważ jednostki mogą stracić przytomność z powodu krytycznych czynności życiowych powstałych w wyniku urazów innych niż śmiertelne.\n W istocie oznacza to, że aby umożliwić zatrzymanie akcji serca dla jednostek AI, to ustawienie musi być włączone.</Polish>
```

File: addons/medical_treatment/stringtable.xml [hunk 11]
```diff
@@ -94,6 +103,7 @@
             <Portuguese>Ativa a manipulação avançada de medicações. Também permite o uso de Adenosina e Atropina.</Portuguese>
             <Chinese>是否擴展藥物管控使其更深度化。並且，啟用腺苷以及阿托品的使用次數</Chinese>
             <Czech>Povoluje hlubší a rozšířené zacházení s léčivy. Také aktivuje Adenosin a Atropin.</Czech>
+            <Polish>Umożliwia dłuższą, bardziej skomplikowaną obsługę leków. Umożliwia także stosowanie adenozyny i atropiny.</Polish>
```
Comment (veteran29) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Polish>Włącza rozszerzoną, bardziej skomplikowaną obsługę leków. Umożliwia także stosowanie adenozyny i atropiny.</Polish>
```

File: addons/medical_treatment/stringtable.xml [hunk 12]
```diff
@@ -198,12 +215,13 @@
...
             <Chinese>要受過何種程度的醫療訓練才可以使用腎上腺素</Chinese>
             <Czech>Úroveň výcviku pro použití epinefrinu</Czech>
+            <Polish>Poziom wyszkolenia wymagany do korzystania z epinefryny.</Polish>
         </Key>
         <Key ID="STR_ACE_Medical_Treatment_LocationEpinephrine_DisplayName">
             <English>Locations Epinephrine</English>
             <German>Orte für Epiniphrin</German>
             <Spanish>Ubicaciones epinefrina</Spanish>
-            <Polish>Ograniczenia adrenaliny</Polish>
+            <Polish>Miejsca użycia adrenaliny</Polish>
```
Comment (veteran29) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Polish>Miejsca użycia epinefryny</Polish>
```

File: addons/medical_treatment/stringtable.xml [hunk 13]
```diff
@@ -281,6 +302,7 @@
             <Portuguese>KPS Descartável</Portuguese>
             <Chinese>個人急救包為消耗品</Chinese>
             <Czech>Spotřebuj osobní lékárničku (PAK)</Czech>
+            <Polish>Utrata apteczek osobistych</Polish>
```
Comment (veteran29) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Polish>Zużycie apteczek osobistych</Polish>
```

File: addons/medical_treatment/stringtable.xml [hunk 14]
```diff
@@ -290,6 +312,7 @@
             <Portuguese>Controla se o KPS deve ser descartado/consumido após o uso.</Portuguese>
             <Chinese>設定個人急救包是否為消耗品</Chinese>
             <Czech>Nastavuje zda má být osobní lékárnička (PAK) spotřebována po použití.</Czech>
+            <Polish>Kontroluje, czy apteczka osobista powinna być stracona po użyciu.</Polish>
```
Comment (veteran29) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Polish>Kontroluje, czy apteczka osobista powinna być zużyta po użyciu.</Polish>
```

File: addons/medical_treatment/stringtable.xml [hunk 15]
```diff
@@ -353,6 +382,7 @@
             <Portuguese>Kit Cirúrgico Consumível</Portuguese>
             <Chinese>手術包為消耗品</Chinese>
             <Czech>Spotřebuj sešívací sadu</Czech>
+            <Polish>Utrata Zestawów Chirurgicznych</Polish>
```
Comment (veteran29) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Polish>Zużycie Zestawów Chirurgicznych</Polish>
```

File: addons/medical_treatment/stringtable.xml [hunk 16]
```diff
@@ -362,6 +392,7 @@
             <Portuguese>Controla se o Kit Cirúrgico deve ser descartado/consumido após o uso.</Portuguese>
             <Chinese>設定手術包是否為消耗品</Chinese>
             <Czech>Nastavuje </Czech>
+            <Polish>Kontroluje, czy Zestaw Chirurgiczny powinien być stracony po użyciu.</Polish>
```
Comment (veteran29) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Polish>Kontroluje, czy Zestaw Chirurgiczny powinien być zużyty po użyciu.</Polish>
```

File: addons/medical_treatment/stringtable.xml [hunk 17]
```diff
@@ -457,6 +496,7 @@
             <Portuguese>Duração de Lixo</Portuguese>
             <Chinese>醫療用廢棄物時長</Chinese>
             <Czech>Životnost odpadků</Czech>
+            <Polish>Żywotność Śmieci</Polish>
```
Comment (veteran29) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Polish>Czas trwania Śmieci</Polish>
```

File: addons/medical_treatment/stringtable.xml [hunk 18]
```diff
@@ -466,6 +506,7 @@
             <Portuguese>Controla o tempo de vida de objetos de lixo criados em segundos. -1 é para sempre.</Portuguese>
             <Chinese>控制醫療用廢棄物的時長，以秒為單位。設定為-1則是永不刪除。</Chinese>
             <Czech>Nastavuje za jak dlouho zdravotnické odpadky zmizí. -1 je navždy.</Czech>
+            <Polish>Kontroluje żywotność śmieci w sekundach. -1 pozostawia je na zawsze.</Polish>
```
Comment (veteran29) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Polish>Kontroluje czas trwania śmieci w sekundach. -1 pozostawia je na zawsze.</Polish>
```

File: addons/medical_treatment/stringtable.xml [hunk 19]
```diff
@@ -917,7 +968,7 @@
             <Russian>Жгут</Russian>
             <Spanish>Torniquete (CAT)</Spanish>
             <French>Garrot (CAT)</French>
-            <Polish>Staza (typ. CAT)</Polish>
+            <Polish>Staza taktyczna (typu CAT)</Polish>
```
Comment (veteran29) [inline addons/medical_treatment/stringtable.xml]: Inconsistent with earlier translation
```suggestion
            <Polish>Staza taktyczna (CAT)</Polish>
```

File: addons/medical_treatment/stringtable.xml [hunk 20]
```diff
@@ -3636,6 +3696,7 @@
             <Chinese>%1 已套用固定板</Chinese>
             <Czech>%1 použil dlahu</Czech>
             <Japanese>%1 は添え木を当てた</Japanese>
+            <Polish>% 1 założył szynę</Polish>
```
Comment (veteran29) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Polish>%1 założył szynę</Polish>
```

**Comment by PabstMirror** [comment]
I can handle merge conflicts

**Comment by PabstMirror** [comment]
Merge conflicts fixed,
if you are using a git client, make sure to `pull` first before continuing work

File: addons/cookoff/stringtable.xml (hunk 1 above)
Comment (Keewayz) [reply addons/cookoff/stringtable.xml]: +1

File: addons/csw/stringtable.xml (hunk 2 above)
Comment (Keewayz) [reply addons/csw/stringtable.xml]: +1

File: addons/csw/stringtable.xml (hunk 3 above)
Comment (Keewayz) [reply addons/csw/stringtable.xml]: +1

File: addons/medical_blood/stringtable.xml (hunk 4 above)
Comment (Keewayz) [reply addons/medical_blood/stringtable.xml]: +1

File: addons/medical_blood/stringtable.xml (hunk 5 above)
Comment (Keewayz) [reply addons/medical_blood/stringtable.xml]: +1

File: addons/medical_treatment/stringtable.xml (hunk 16 above)
Comment (Keewayz) [reply addons/medical_treatment/stringtable.xml]: +1

File: addons/medical_treatment/stringtable.xml (hunk 17 above)
Comment (Keewayz) [reply addons/medical_treatment/stringtable.xml]: +1

File: addons/medical_treatment/stringtable.xml (hunk 18 above)
Comment (Keewayz) [reply addons/medical_treatment/stringtable.xml]: +1

File: addons/medical_treatment/stringtable.xml (hunk 19 above)
Comment (Keewayz) [reply addons/medical_treatment/stringtable.xml]: +1

File: addons/medical_treatment/stringtable.xml (hunk 20 above)
Comment (Keewayz) [reply addons/medical_treatment/stringtable.xml]: +1

**Comment by Keewayz** [comment]
Im okay with all changes made by @veteran29

**Comment by veteran29** [comment]
If you're okay with them you should go to `Files changed` tab and apply the suggestions in batch.

**Comment by veteran29** [comment]
@Keewayz could you add my suggestions so we can get this merged?

**Pull request merged**